tests/recordings/test_ref_createListDelete.yaml

```diff
@@ -0,0 +1,124 @@
+interactions:
```

Comment:
The yaml is collected on a test which was run without the cache cleared. This will fail in recording if the test is run on a system with cache cleared.